judge/widgets/mixins.py

```diff
@@ -34,7 +34,7 @@ class CompressorWidgetMixin(object):
     except ImportError:
         pass
     else:
-        if getattr(settings, 'COMPRESS_ENABLED', not getattr(settings, 'DEBUG', False)):
+        if settings.COMPRESS_ENABLED and not settings.DEBUG:
```

Comment:
No, this also changes semantics because if you do `DEBUG = False`, `COMPRESS_ENABLED` is ignored. The correct solution here is `getattr(settings, 'COMPRESS_ENABLED', not settings.DEBUG)` and removing `COMPRESS_ENABLED` from `settings.py`.